ansible/roles/controller/tasks/deploy.yml

```diff
@@ -154,6 +154,7 @@
 
       "CONFIG_whisk_info_date": "{{ whisk.version.date }}"
       "CONFIG_whisk_info_buildNo": "{{ docker.image.tag }}"
+      "CONFIG_whisk_cluster_name": "{{ whisk.cluster_name | lower }}"
```

Comment:
This is one of the mandatory configs.
This is used as a prefix for the ETCD data.
When we deploy a new version, each version of the components can communicate with each other with based on this information.

Reply:
Can we default this to `whisk` in the `.conf`? That's what is used right now right? Or is it like the kafka prefix where it would be in front of the `whisk` prefix so if it's not defined in config then it just uses `''`

Reply:
If no configuration is provided, the default value `whisk` will be used.
But the problem is if you use different values for each deployment to distinguish them, it wouldn't work as the scheduler data in ETCD only applies the prefix.
Controllers fetch scheduler endpoints with this prefix and schedulers fetch available invokers with this prefix.

Reply:
Yea that makes sense just if you don't care about that you don't have to define this right? If you did one deployment with no configuration it will be whisk and then you could do a new deployment defining the config and then there won't be any clash

Reply:
Exactly.